gnovm/stdlibs/time/timezoneinfo.gno

```diff
@@ -624,39 +631,6 @@ var zoneinfo *string
 
 func loadFromEmbeddedTZData(name string) ([]byte, bool) // injected
 
-// XXX var zoneinfoOnce sync.Once
-
-// LoadLocation returns the Location with the given name.
-//
-// If the name is "" or "UTC", LoadLocation returns UTC.
-// If the name is "Local", LoadLocation returns Local.
-//
-// Otherwise, the name is taken to be a location name corresponding to a file
-// in the IANA Time Zone database, such as "America/New_York".
-//
-// LoadLocation looks for the IANA Time Zone database in the following
-// locations in order:
-//
-// - the directory or uncompressed zip file named by the ZONEINFO environment variable
-// - on a Unix system, the system standard installation location
-// - $GOROOT/lib/time/zoneinfo.zip
-// - the time/tzdata package, if it was imported
-func LoadLocation(name string) (*Location, error) {
```

Comment:
Why remove it?  
This isn't simplification; it's the removal of a feature we expect to function in a well-known standard library package.  
Simplification should focus on making it easier to use or  something you shouldn't have to worry about.

Reply:
Read more about this here
https://github.com/gnolang/gno/pull/3193#discussion_r2023454656